wolfcrypt/test/test.c

```diff
@@ -1129,7 +1149,7 @@ initDefaultName();
         test_pass("mp       test passed!\n");
 #endif
 
-#if defined(WOLFSSL_PUBLIC_MP) && defined(WOLFSSL_KEY_GEN)
+#if defined(WOLFSSL_PUBLIC_MP) && defined(WOLFSSL_KEY_GEN) && !defined(WOLFSSL_OLD_PRIME_CHECK)
```

Comment:
Let's keep this test enabled with `WOLFSSL_OLD_PRIME_CHECK`, but you are welcome to disable it for `WOLFSSL_LINUXKM`.

Reply:
done.